script/README.md

```diff
@@ -30,8 +52,14 @@ Here's how to get your local environment up and running:
 2. run `script/teach-class` and choose option 0 to set up your environment variables. You'll need the following pieces of information from [above](#remote-configuration):
    - the title of your newly created training organization
    - your username, or the username of a shared teaching account
+
+   <!-- textlint-disable terminology -->
+
    - your personal access token (PAT) or the PAT of the shared teaching account that includes the scopes **repo** and **delete_repo**.
    _Note: If your organization has SAML SSO enabled, you would need to [authorize your personal access token for use with SAML SSO](https://docs.github.com/en/github/authenticating-to-github/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)._
+
+   <!-- textlint-enable terminology -->
+
```

Comment:
This textlint rule has been [fixed in version 3.0.1](https://github.com/sapegin/textlint-rule-terminology/issues/38#issuecomment-1143205870) and will be included in the next [super-linter](https://github.com/github/super-linter) release. I’ll open a new PR to remove the rule exceptions/workarounds once that release is available.